Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.Samples.WPF.App/MainWindow.xaml.cs

```diff
@@ -0,0 +1,93 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Toolkit.Win32.UI.Controls.WPF;
+using System;
+using System.Windows;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Win32.Samples.WPF.App
+{
+    /// <summary>
+    /// Interaction logic for MainWindow.xaml
+    /// </summary>
+    public partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            InitializeComponent();
+        }
+
+        private void inkCanvas_Loaded(object sender, RoutedEventArgs e)
+        {
+            inkCanvas.InkPresenter.InputDeviceTypes = CoreInputDeviceTypes.Mouse | CoreInputDeviceTypes.Pen | CoreInputDeviceTypes.Touch;
+        }
+
+        private void inkToolbar_Initialized(object sender, EventArgs e)
```

Comment:
Why empty?

Reply:
To show the events can be handled more than to do anything with them.  I'll add comments to make that more clear and remove the bunch of handlers that crept in that aren't actually getting used.

Reply:
Ok, I'll pull them out...